conference/react-js/src/constants/roles.js

```diff
@@ -1,3 +1,8 @@
 export const speaker = 'ROLE_SPEAKER';
 export const organiser = 'ROLE_ORGANISER';
 export const admin = 'ROLE_ADMIN';
+export const rolesUI = {
+  ROLE_SPEAKER: 'speaker',
```

Comment:
I think you can write `[speaker]` here. and for next props the same. It will help to stay up-to-date with current names